ui/src/main/AndroidManifest.xml

```diff
@@ -37,7 +37,8 @@
 
     <application
         android:name=".Application"
-        android:allowBackup="false"
+        android:allowBackup="true"
+        android:fullBackupContent="@xml/backup"
```

Comment:
This only works for API ≤30. For API ≥31, you need android:dataExtractionRules and a different file. See https://developer.android.com/identity/data/autobackup for details.